#goal definition
string dock_tf_name
---
#result definition
bool success
---
#feedback
float32 distance